src/routes/(modeles-services)/_common/display/service-mobilisation.svelte

```diff
@@ -10,7 +10,7 @@
   import SharingModal from "./modals/sharing-modal.svelte";
 
   export let service: Service;
-  export let contactBoxOpen = false;
+  export let contactBoxOpen = !(service.contactEmail || service.contactPhone);
```

Comment:
J'ajouterais un commentaire de ce type :
```suggestion
  // Si il n'y a pas d'information de contact, on n'affiche pas le bouton
  export let contactBoxOpen = !(service.contactEmail || service.contactPhone);
```